### PR TITLE
Better translation of "authorization"

### PR DESCRIPTION
**Pull request by piotrdrag**
From the calque "autoryzować" to more Polish "upoważniać".

### CODE REVIEW

File: security/manager/chrome/pipnss/nsserrors.properties [hunk 1]
```diff
@@ -133,7 +133,7 @@ SSL_ERROR_NEXT_PROTOCOL_NO_CALLBACK=Rozszerzenie negocjacji następnej wersji pr
 SSL_ERROR_NEXT_PROTOCOL_NO_PROTOCOL=Serwer nie obsługuje żadnego z protokołów ogłaszanych przez klienta w rozszerzeniu ALPN.
 SSL_ERROR_INAPPROPRIATE_FALLBACK_ALERT=Serwer odrzucił żądanie, ponieważ klient zmienił wersję TLS na niższą niż obsługiwana przez serwer.
 SSL_ERROR_WEAK_SERVER_CERT_KEY=Certyfikat serwera zawiera klucz publiczny o zbyt słabym szyfrowaniu.
-SEC_ERROR_IO=Podczas autoryzacji bezpieczeństwa wystąpił błąd wejścia/wyjścia.
+SEC_ERROR_IO=Podczas upoważniania bezpieczeństwa wystąpił błąd wejścia/wyjścia.
```
Comment (splewako) [inline security/manager/chrome/pipnss/nsserrors.properties]: This one is incorrect but I'm not sure how it should be fixed - one option would be to remove "bezpieczeństwa" maybe.

Comment (piotrdrag) [inline security/manager/chrome/pipnss/nsserrors.properties]: Or "Podczas upoważniania zabezpieczeń wystąpił błąd wejścia/wyjścia."?

Comment (splewako) [inline security/manager/chrome/pipnss/nsserrors.properties]: I don't think it would be correct to "upoważnić" a thing or a concept.

Comment (piotrdrag) [inline security/manager/chrome/pipnss/nsserrors.properties]: I don't know. I've removed "bezpieczeństwa".

**Comment by splewako** [comment]
https://hg.mozilla.org/l10n-central/pl/rev/201e6be0e20d and https://github.com/aviarypl/mozilla-l10n-central-pl/commit/4d615ef65d9ed0e93831cbab126e03752efb6c0e

Thank you for the patch

**Pull request closed without merging**